velox/expression/CastExpr.cpp

```diff
@@ -135,13 +135,13 @@ VectorPtr applyDecimalToDoubleCast(
   auto sourceVector = input.as<SimpleVector<TInput>>();
   auto resultBuffer =
       result->asUnchecked<FlatVector<double>>()->mutableRawValues();
-  auto toPrecisionScale = getDecimalPrecisionScale(*fromType);
+  auto [precision, scale] = getDecimalPrecisionScale(*fromType);
+  const auto simpleInput = input.as<SimpleVector<TInput>>();
+  const auto kDenominator = (double)DecimalUtil::kPowersOfTen[scale];
```

Comment:
naming: kXxx is for static constants

kDenominator -> denominator

Reply:
my bad. Thanks. Fixed.